x-pack/plugins/canvas/public/components/workpad_header/control_settings/kiosk_controls.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  EuiDescriptionList,
+  EuiDescriptionListDescription,
+  EuiDescriptionListTitle,
+  EuiFormLabel,
+  EuiHorizontalRule,
+  EuiLink,
+  EuiSpacer,
+  EuiSwitch,
+  EuiText,
+  EuiFlexGrid,
+  EuiFlexItem,
+} from '@elastic/eui';
+import { timeDurationString } from '../../../lib/time_duration';
+import { CustomInterval } from './custom_interval';
+
+const ListGroup = ({ children }) => <ul style={{ listStyle: 'none', margin: 0 }}>{[children]}</ul>;
```

Comment:
It's interesting you have `RefreshItem` in scope and this as global.  I'd relocate the other here to avoid re-renders.

Reply:
`RefreshItem` uses a prop. I could hoist it via a partial, but either way the location of the helper shouldn't affect re-renders. What am I missing?